server/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
+  <sessions-enabled>true</sessions-enabled>
+  <runtime>java8</runtime>
+  <static-files>
+    <!-- prevent unwanted caching when accessing via the web preview server -->
+    <include path="/**" expiration="0s" />
```

Comment:
I don't think you want /**. This will probably expose every file in the codebase, including things like Java files. Since this is open source anyway that's probably not a security hole but still not a necessary thing to expose.

Reply:
Oh I see, this is only applying to the /webapp folder. Have you tried this on an actual appengine instance yet?